Managed Services/getting-started-with-managed-wordpress.md

```diff
@@ -0,0 +1,30 @@
+{{{
+  "title": "Getting Started with Managed Wordpress",
+  "date": "04-30-2015",
+  "author": "Bill Burge",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+**Overview**
+
+[WordPress](http://www.wordpress.org "WordPress.org") is a free open-source blogging tool and content management system (CMS) based on PHP and MySQL.
+
+CenturyLink Clouds solution is an Enterprise class, cloud-based solution for enterprise level customers.
+
+blah blah blah...........................
```

Comment:
the master branch is meant to be able to be deployed to production at any point so no "placeholder" text can make it into the master branch